.gitignore

```diff
@@ -4,4 +4,6 @@
 *.iml
 .env
 database.db
-/tags-repo
\ No newline at end of file
+/tags-repo
```

Comment:
I forgot about this. This can be removed lol

Reply:
Any particular reason this was not touched

Reply:
?

Reply:
You resolved this, and the other conversation with out response

Reply:
Removed